File: docs/article/source/csharp/apps/cargo-tracker/src/CargoTracker.Web/Booking/Domain/Model/CargoType.cs
namespace CargoTracker.Booking.Domain.Model;

/// <summary>
/// 貨物種別。Estimation にも同名 enum があるが、BC 独立を優先して Booking 独自に定義する。
/// 共通化は CargoType の変更頻度が見えた後に ADR で判断する。
/// </summary>
public enum CargoType
{
    General,
    Hazardous,
    Refrigerated,
}

File: apps/cargo-tracker/backend/bookingms/src/main/java/com/example/cargotracker/booking/domain/model/valueobjects/CargoType.java
package com.example.cargotracker.booking.domain.model.valueobjects;

/** 貨物種別（domain-model.md「ユビキタス言語」）。 */
public enum CargoType {
    /** 一般。 */
    GENERAL,
    /** 危険物。危険物申告が要る。 */
    HAZARDOUS,
    /** 冷凍・冷蔵。温度管理条件が要る。 */
    REFRIGERATED
}
